hackvote/test/fixtures/projects.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+  creators: MyString
+  event_id: 
```

Comment:
Umm, I don't think I actually ended up using these in tests... that might be why...